src/pyodide/BUILD.bazel

```diff
@@ -17,24 +17,37 @@ capnp_embed(
 )
 
 copy_file(
-    name = "packages_capnp_file",
-    src = "pyodide_packages.capnp",
-    out = "generated/pyodide_packages.capnp",
+    name = "python_entrypoint_file",
+    src = "python-entrypoint.js",
+    out = "generated/python-entrypoint.js"
 )
 
 capnp_embed(
-    name = "packages_capnp_file_embed",
-    src = "generated/pyodide_packages.capnp",
-    deps = ["packages_capnp_file"],
+    name = "python_entrypoint_file_embed",
+    src = "generated/python-entrypoint.js",
+    deps = ["python_entrypoint_file"],
+)
+
+copy_file(
+    name = "pyodide_extra_capnp_file",
+    src = "pyodide_extra.capnp",
+    out = "generated/pyodide_extra.capnp",
+)
+
+capnp_embed(
+    name = "pyodide_extra_file_embed",
+    src = "generated/pyodide_extra.capnp",
+    deps = ["pyodide_extra_capnp_file"],
```

Comment:
A lot of duplication here with the other files that are embedded. Can we refactor this into a function to make it easier to see which files are getting embedded and with what names?

Reply:
Sure, how about in a followup?

Reply:
Sure. Big fan of leaving things like this to other PRs so we can move fast :)